cfgov/diversity_inclusion/views.py

```diff
@@ -0,0 +1,28 @@
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+from forms import VoluntaryAssessmentForm
+
+
+class GetName(TemplateView):
```

Comment:
Could this be simplified with use of a [`FormView`](https://docs.djangoproject.com/en/1.11/ref/class-based-views/generic-editing/#formview) instead, something like:

```py
class GetName(FormView):
    template_name = 'diversity_inclusion/voluntary-assessment-form.html'
    form_class = VoluntaryAssessmentForm
    success_url = reverse_lazy('diversity_inclusion:form_submitted')
```

Reply:
Thanks for the suggestion! Changed in https://github.com/cfpb/cfgov-refresh/pull/5265/commits/a82f1df9e65bb0be103c0aeecfe68b70229b1de1.